### DISCUSSION TITLE
Color picker doesn't work

### DISCUSSION THREAD

**Discussion by Dacostaphexe** [Mahmud0808/ColorBlendr#96] [Q&A]
ok, in one ui 6.1 there were problems with the selection of colors based on the wallpaper, where these colors were not as connected with what I wanted. As soon as I found out about the app, I started trying to use it, but it didn't work. so I left it aside, but when changing the wallpaper the colors remain the same as before downloading the app. detail that I use shizuku because it's a new cell phone, Samsung a35, and I won't root it. I'm looking for a solution, mainly for the app to run on OneUI or at least return the default color picker. I also saw that there is no way to change these colors in the developer options. Does anyone have any tips?

**Comment by Mahmud0808** [comment]
Use Theme Park for changing OneUI system colors. And use ColorBlendr for changing google apps' colors. ColorBlendr can't change OneUI system accent colors because it doesn't use Material You colors.
